### DISCUSSION TITLE
Ignore specific middleware

### DISCUSSION THREAD

**Discussion by Pennywise96** [filamentphp/filament#13098] [Help]
### Package

Panel builder

### Package Version

v3

### How can we help you?

Hi!
Im build at the moment a app where one part is only filementphp (backend) and a other part is only livewire (frontend).
in the frontend i need to apply a middleware to the update routes of livewire:

`
Livewire::setUpdateRoute(function ($handle) {

      return Route::post('/livewire/update', $handle)->middleware(['setPage']);
});
`

But when I do this, filament also uses this middleware and this leads to errors on update actions.
So is it possible to remove or ignore a specific middleware with filament? Or is there a way to isolate both parts of my app?

Thanks!

**Comment by leandrocfe** [comment]
I don't think we have a `withoutMiddleware` option. However, I guess you might be able to use filament()->getCurrentPanel() within the middleware to check if it’s running in the panel.

**Reply by Pennywise96** [reply]
Its not directly in the middleware of filament, its on the /livewire/update route of Livewire. I need that middleware in the frontend. But when i write this in my ServiceProvider:

```
Livewire::setUpdateRoute(function ($handle) {
                return Route::post('/livewire/update', $handle)
                   ->middleware(['setPage']);
            });
```

The middleware 'setPage' gets also applied when i update a record in filament and this leads to errors. But i dont know if its even possible to set the middleware only on one part of my app...

**Reply by leandrocfe** [reply]
What are you doing in setPage?

**Comment by ignacio-dev** [comment]
I need this as well, did you find a way?
